interface/usergroup/mfa_totp.php

```diff
@@ -185,6 +185,8 @@ function docancel() {
                                             <br />
                                             <img src="<?php echo attr($qr); ?>" class="img-responsive center-block" style="height:200px !Important"/>
                                             <br />
+                                            <p><?php echo xlt("Or paste in the following code into your authenticator app"); ?></p>
+                                            <p><?php echo $mfaAuth->getSecret(); ?></p>
```

Comment:
also should do at https://github.com/openemr/openemr/blob/master/setup.php#L1377-L1404
(this allows setting up mfa on the user during openemr installation)

Reply:
@bradymiller Thanks for catching that.  Added to the setup.

Reply:
Yep I call him ole Eagle Eye! 😉